.github/workflows/bahmni_metabase_build_publish.yaml

```diff
@@ -13,9 +13,11 @@ jobs:
       - name: Trivy Scan
         run: ./.github/trivy_scan.sh
 
-  docker-build-publish:
+  build-publish-docker-helm:
     name: Docker Build & Publish
```

Comment:
Please include helm also here in the name

Reply:
Yes Done , Thank you